src/main/java/data/DatabaseQuery.java

```diff
@@ -0,0 +1,62 @@
+package data;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.TreeSet;
+
+/* Returns all files id and names with the purpose of having them show to webpage. */
+public class DatabaseQuery {
+  /* Returns all files id and names with the purpose of having them show to webpage. */
+  private static ArrayList<String> getAllFiles(Database database) {
+    return database.getAllFiles();
+  }
+
+  // returns json format of database entries
+  public static String getJson(Database database) {
+
+    // Example of sample Json format ~
+    /*
+     * [data: {runId: aks;ldfja entries: []}]
+     */
+
+    String fileName = database.getDatabaseName() + "/";
+    fileName.trim();
+    /*System.out.println("The filename is " + fileName);
+    File dir = new File(fileName);
+    System.out.println(dir.getAbsolutePath());
+    */
+    ArrayList<String> filenames = getAllFiles(database);
+    if (filenames == null) {
+      System.out.println("Directory is empty");
+      return "{}";
+    }
```

Comment:
Alternately, you could define `getAllFiles` to never return null. Returning an empty `ArrayList` when there are no files is sensible, and means that the caller doesn't have to do this null check.